src/picongpu/include/plugins/output/GatherSlice.hpp

```diff
@@ -188,6 +188,7 @@ struct GatherSlice
                 insertData(dstBox, srcBox, head->node.offset, head->node.maxSize);
             }
 
+            __deleteArray(fullData);
```

Comment:
not yet, but if you want you can create a PR.
also, the macros need a little update, see #532